incidents/2020-05-20-seach-outage.md

```diff
@@ -0,0 +1,66 @@
+# 2020-05-20 Search Outage
+
+**Incident Leader: Giorgio Sironi, Luke Skibinski**
+
+## Description
+
+Summary of the findings of the investigation/post-mortem.
+
+## Timeline
+
+**2020-05-15**
+
+- Suspected date the issue started
+
+**2020-05-19**
+
+- 17:57 Peter R posted a message on the sitefeedback Slack channel that some articles weren't showing on the site.
+
+**2020-05-20**
+
+- 08:36 Peter R posted a message on the features-production Slack channel that some articles weren't showing on the site.
+
+- 08:55 Fred A raised issue in GitHub [#5661](https://github.com/elifesciences/issues/issues/5661) that articles where not appearing.
+
+- 09:11 Mael reached out to Nathan for help via Slack and GitHub
+
+- 09:15 Mael raised issue in standup
+
+- 09:42 Nathan looked at the GitHub issue, identied that it might be related to search but said he wouldn't be able to look into the problem until the afternoon.
+
+- 17:31 Incident Response channel on slack, message posted by Rowena about eToc being sent out with no content.
+
+- 17:40 Giorgio started investigation
+
+**2020-05-21**
+
+- 05:07 Luke S notes that the issue is now resolved
```

Comment:
https://github.com/elifesciences/issues/issues/5664#issuecomment-632756145 was a separate problem which I'd consider out of the scope of this incident.